api-reference/50 Common/Object Structures/PdfExportDataGridProps/onRowExporting.md

```diff
@@ -4,16 +4,207 @@ type: function(options)
 ---
 ---
 ##### shortDescription
-<!-- Description goes here -->
+A function that is executed before the row of the DataGrid is exported. Allows you to customize the height of the exported row.
 
 ##### param(options): Object
-<!-- Description goes here -->
+Information about the event.
 
 ##### field(options.rowCells): Array<PdfCell>
-<!-- Description goes here -->
+Cells for the currently exporting row.
 
 ##### field(options.rowHeight): Number
+Allows you to specify the minimum height of the exported row. If the specified value is less than the height of the text, the row height will be automatically increased.
```

Comment:
If the specified value is less than the height of the text, the component increases the row height automatically.